dss.go

```diff
@@ -0,0 +1,123 @@
+package globalsign
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/hex"
+	"strings"
+	"time"
+)
+
+// DSSService .
```

Comment:
broken comment...
Do we need this interface?  Where is it implemented?

Reply:
Updated at a3ddc191292d0952b56e2269c3cbfa6f517b98fa